CHANGELOG.md

```diff
@@ -34,6 +34,8 @@ RELEASING:
  -->
 
 ## [Unreleased]
+### Fixed
```

Comment:
New features should go under
`### Added`
as this is not a fix but an enhancement

e.g. 
```
### Added
- possibility to pass single `radiuses`-parameter value for all waypoints ( and the rest)
```

Reply:
In fact, it is not - where previously a request with more than one coordinate and only one radius value would yield a parameter value/format error, it now doesn't anymore.
The original commit's message states that as well - this is not backwards compatible ;)

Reply:
> where previously a request with more than one coordinate and only one radius value would yield a parameter value/format error, it now doesn't anymore.

But i thought that is exactly the added feature, no?

Current documenation states:
![radiuses](https://user-images.githubusercontent.com/23240110/119196666-d19c5680-ba86-11eb-821b-3b739632ff25.JPG)

so an error should be expected if the radiuses count currently doesn't match the coordinate count.

also https://github.com/GIScience/openrouteservice/issues/923 is labeled as enhancement | feature

I don't see a reason to put it in `Fixed` category 🤷